lib/kaffe/group_member/worker/worker_manager.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Kaffe.WorkerManager do
+  @moduledoc """
+  This is the main process for launching group members and workers.
+
+  The process begins by starting the client connection to Kafka. Then, group
+  members are created for each of the configured topics.
+
+  This is also the place where subscribers get the PID for a Worker.
+  """
+  
+  use GenServer
```

Comment:
If this module is responsible for creating group members and workers, why isn't it a supervisor?